### ISSUE TITLE
Echo + excluded apps list

### ISSUE THREAD

**Issue by QBANIN** [ThePBone/RootlessJamesDSP#157]
Hi! When I use this car navigation https://play.google.com/store/apps/details?id=pl.naviexpert.market every turn announcement has echo. Once I disable RootlessJamesDSP issue is gone. Tried to add this app to excluded list but... It is not listed in the apps to exlude list... Why? I'm using latest 1.6.3 DSP version.

**Comment by ThePBone** [comment]
I received recently an (unjustified) policy violation from Google Play because I used the Android permission to list all apps on the phone. They deemed my use case not important enough to allow me to keep using that permission so I had to remove it. They're very strict about that for some reason.

I'll need to find a workaround for the Play Store version, but for now, it has limited permission to view other installed apps on the device.

The F-Droid version is not affected by this issue: https://f-droid.org/en/packages/me.timschneeberger.rootlessjamesdsp/
